core/src/Instance/AdfsAuthority.cs

```diff
@@ -66,28 +62,10 @@ protected override async Task<string> GetOpenIdConfigurationEndpointAsync(string
         {
             if (ValidateAuthority)
             {
-                DrsMetadataResponse drsResponse = await GetMetadataFromEnrollmentServerAsync(userPrincipalName, requestContext).ConfigureAwait(false);
-                
-                if (!string.IsNullOrEmpty(drsResponse.Error))
-                {
-                    CoreExceptionFactory.Instance.GetServiceException(
-                        drsResponse.Error,
-                        drsResponse.ErrorDescription, 
-                        ExceptionDetail.FromDrsResponse(drsResponse));
-                }
-
-                if (drsResponse.IdentityProviderService?.PassiveAuthEndpoint == null)
-                {
-                    throw CoreExceptionFactory.Instance.GetServiceException(
-                        CoreErrorCodes.MissingPassiveAuthEndpoint,
-                        CoreErrorMessages.CannotFindTheAuthEndpont, 
-                        ExceptionDetail.FromDrsResponse(drsResponse));                        
-                }
-
                 string resource = string.Format(CultureInfo.InvariantCulture, CanonicalAuthority);
                 string webfingerUrl = string.Format(CultureInfo.InvariantCulture,
-                    "https://{0}/adfs/.well-known/webfinger?rel={1}&resource={2}",
-                    drsResponse.IdentityProviderService.PassiveAuthEndpoint.Host,
+                    "https://{0}/.well-known/webfinger?rel={1}&resource={2}",
```

Comment:
>https://{0}/.well-known/webfinger?rel={1}&resource={2}", [](start = 21, length = 56)

is this backwards compatible?

Reply:
There will only be compatability with Adfs 2019...older versions do not support the resource/scope format

Reply:
hmm, why not have support for previous versions? Almost no-one uses ADFS 2019 at this point in time. At least 2016 should be supported imo.

---
In reply to: [223864165](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1293#discussion_r223864165) [](ancestors = 223864165)

Reply:
I think support for 2016 is possible. I'd like to get these changes in ASAP (next couple of days) as frequent updates the library have made confilct resolution very painful, so maybe do 2016 compatability as a separate PR?